<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LPTIN-24Z3GG3" failures="0" tests="6" name="com.StudentPortal.StudentTests.StudentLoginTest1" time="4.978" errors="1" timestamp="2023-08-01T16:56:28 IST" skipped="5">
  <testcase classname="com.StudentPortal.StudentTests.StudentLoginTest1" name="beforeTestMethod" time="3.847">
    <error message="Cannot invoke &quot;org.openqa.selenium.WebDriver.get(String)&quot; because &quot;this.driver&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebDriver.get(String)" because "this.driver" is null
at com.framework.core.AutomationTestPlan.beforeTestMethod(AutomationTestPlan.java:97)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:340)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:294)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:683)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:510)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </error>
  </testcase> <!-- beforeTestMethod -->
  <testcase classname="com.StudentPortal.StudentTests.StudentLoginTest1" name="TC938_2LedgerCardAdjustTransactionDelete" time="0.241">
    <skipped/>
  </testcase> <!-- TC938_2LedgerCardAdjustTransactionDelete -->
  <testcase classname="com.StudentPortal.StudentTests.StudentLoginTest1" name="TC940_1LedgerCardPrint" time="0.203">
    <skipped/>
  </testcase> <!-- TC940_1LedgerCardPrint -->
  <testcase classname="com.StudentPortal.StudentTests.StudentLoginTest1" name="TC940_2LedgerCardPrintDetail" time="0.606">
    <skipped/>
  </testcase> <!-- TC940_2LedgerCardPrintDetail -->
  <testcase classname="com.StudentPortal.StudentTests.StudentLoginTest1" name="TC941_2LedgerCardApplyCredit" time="0.081">
    <skipped/>
  </testcase> <!-- TC941_2LedgerCardApplyCredit -->
  <testcase classname="com.StudentPortal.StudentTests.StudentLoginTest1" name="TC945_2LedgerCardScheduleaRefund" time="0.000">
    <skipped/>
  </testcase> <!-- TC945_2LedgerCardScheduleaRefund -->
</testsuite> <!-- com.StudentPortal.StudentTests.StudentLoginTest1 -->
